README.md

```diff
@@ -0,0 +1,27 @@
+
+[![codeship](https://codeship.com/projects/3b22b880-f94e-0133-aa46-6ecdbb95c520/status?branch=master)](https://codeship.com/projects/151202)
+[![Coverage Status](https://coveralls.io/repos/github/otis-bot/otis-backend/badge.svg?branch=master)](https://coveralls.io/github/otis-bot/otis-backend?branch=master)
+
+
+Otis Backend
+==============================================
+
+Otis Backend is the backend portion of the Otis-Bot. The Backend receives URL's
+from the Otis-Bot and stores them in a server. Otis backend provides an API for
+ a frontend application allowing voting and commenting on the URLs.
+
+##Installing Dependencies
+
+First, have rails and bundler installed locally.
+After cloning to a local repository, you should be able to type `bundle install`
+from the root.
+
+##Testing
+
+After installing the required dependencies (please see above), type
+`bundle exec rspec` from the root.
```

Comment:
It's not a big deal, but I would recommend using `bin/rspec` instead of `bundle exec rspec`. It is shorter to type and uses Spring so it will run faster.